src/tools/tidy/src/features.rs

```diff
@@ -327,7 +327,7 @@ fn map_lib_features(base_src_path: &Path,
             }
             becoming_feature = None;
             if line.contains("rustc_const_unstable(") {
-                // `const fn` features are handled specially.
+                // Const fn features are handled specially.
```

Comment:
Okay, happy to undo!

Reply:
On second thought, I ran a search, and it seems "const fn" without the quotes is much more common in comments. I think it's okay personally, since "fn" can be considered an abbreviation for "function", and "const" and attribute/adjective.

Reply:
(But I'll still revert it if you prefer.)

Reply:
I'm not entirely sure that commonality is the right argument here. I don't think that either of these is particularly bad, but given that it's a specific rust feature, and that's it's name, `const fn` seems like its "proper name". Dunno.